apps/codeforafrica/src/components/ImpactCard/ImpactCard.js

```diff
@@ -7,13 +7,20 @@ import React from "react";
 
 const ImpactCardRoot = styled(Card, {
   slot: "Root",
-})(() => ({
-  maxWidth: "350px",
+})(({ theme: { breakpoints } }) => ({
+  width: "100%",
   boxShadow: "none",
+
+  [breakpoints.up("md")]: {
+    width: "326px",
+  },
+  [breakpoints.up("lg")]: {
+    width: "332px",
+  },
 }));
 
-const ImpactCard = React.forwardRef(function ImpactCard(props, ref) {
-  const { image, title, value, description } = props;
+const ImpactCard = React.forwardRef(function ImpactCard({ initiative }, ref) {
+  const { image, title, value, description } = initiative;
```

Comment:
Let card receive `image`, `title`, `value`, `description` directly rather than receive `initiative` and then pulling individual props from it.